Readme.md

```diff
@@ -2,6 +2,16 @@
 
 This project is working in progress.
 
+## Stack
+
+* [VueJs](https://vuejs.org/)
+* [TypeScipt](https://www.typescriptlang.org/)
+
+
 ## Contribute
 
-run `yarn` to install all the dependencies.
\ No newline at end of file
+run `yarn` to install all the dependencies.
+run `npx depcheck` to see unused dependencies. [you_Can_Uninstall_Them](https://www.freecodecamp.org/news/npm-uninstall-how-to-remove-a-package/#:~:text=To%20remove%20a%20dev%20dependency%2C%20you%20need%20to%20attach%20the,the%20name%20of%20the%20package.&text=You%20must%20run%20the%20command,where%20the%20dependency%20is%20located.
+)
```

Comment:
Unfortunately, this command seems mark `eventemitter3` as unused dependency. Besides that, `npx depcheck` is more than a maintainer only hint, which is useless to normal developers. I'm afraid this can't be merged before the command got carefully evaluated.

Reply:
owh, yes sorry about that. :-)

Reply:
Anyway, thanks for opening a PR. If you like you could give a new proposal or just delete those which couldn't be merged, and I'll merge the rest.